src/resolvers/Chat/admins.ts

```
@@ -1,14 +1,21 @@
-import { User } from "../../models";
-import type { ChatResolvers } from "../../types/generatedGraphQLTypes";
+import { InterfaceUser, User } from "../../models";
```

ESLint

[error] Imports "InterfaceUser" are only used as type.

(@typescript-eslint/consistent-type-imports)